imagedephi/redact/redact.py

```diff
@@ -66,6 +67,7 @@ def redact_images(
     except PermissionError:
         click.echo("Could not redact images, invalid output directory. Choose a writable directory")
         sys.exit()
+    show_redaction_plan(input_path)
```

Comment:
Scaled back to this simple solution in light of pending logger work. My original thought and attempt was to have a new prompt for each image that didn't have a plan, however knowing now that a user is likely to run in a batch I think simple reporting is the proper way to go.

As a follow up we should likely have a way to hand specific images instead of just a directory so that a user can feed in the images that are reported as not supported here. Filed corresponding issue: #121